packages/react-components/src/templates/TeamProfileHeader.tsx

```diff
@@ -28,6 +28,18 @@ const containerStyles = css({
   padding: `${36 / perRem}em ${contentSidePaddingWithNavigation(10)} 0`,
 });
 
+const titleStyle = css({
+  display: 'flex',
+  flexFlow: 'row',
+  gap: `${15 / perRem}em`,
+  alignItems: 'center',
+  [`@media (max-width: ${tabletScreen.width - 1}px)`]: {
```

Comment:
Generally we take a mobile first approach and then override styles with breakpoints for larger devices

Reply:
Nice, I've changed it here https://github.com/yldio/asap-hub/pull/2112/commits/2de1fcb0c6efc5bc84a5db2c551910e2ad634804